tensorflow_probability/examples/cifar10_bnn.py

```diff
@@ -0,0 +1,263 @@
+# Copyright 2018 The TensorFlow Probability Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""Trains a Bayesian neural network to classify CIFAR-10 images.
+
+The architecture can be either ResNet [1] or VGG [2].
+
+#### References
+
+[1]: He, Kaiming, Xiangyu Zhang, Shaoqing Ren, and Jian Sun.
+     "Deep residual learning for image recognition."
+     _Proceedings of the IEEE_, 2016.
+     https://arxiv.org/abs/1512.03385
+[2]: Simonyan, Karen, and Andrew Zisserman.
+     "Very deep convolutional networks for large-scale image recognition."
+     arXiv preprint arXiv:1409.1556 (2014).
+     https://arxiv.org/pdf/1409.1556.pdf
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+# Dependency imports
+from absl import flags
+import matplotlib
+matplotlib.use("Agg")
+import numpy as np
+import tensorflow as tf
+
+from tensorflow.python.keras.datasets import cifar10
+
+from models.bayesian_resnet import bayesian_resnet
+from models.bayesian_vgg import bayesian_vgg
+
+import warnings
+warnings.simplefilter(action='ignore')
+
+
+tfd = tf.contrib.distributions
```

Comment:
Use `import tensorflow_probability as tfp` above, and then write here `tfd = tfp.distributions`.